helm/dagster/values.yaml

```diff
@@ -14,6 +14,10 @@ global:
   # dagster subcharts will not create their own service accounts.
   serviceAccountName: ""
 
+  # You can specify a secret name that supplies the broker URL (in a
+  # DAGSTER_CELERY_BROKER_URL key) and backend URL (in a DAGSTER_CELERY_BACKEND_URL key)
+  celeryConfigSecretName: "dagster-celery-config-secret"
```

Comment:
Most k8s resources that we create are scoped by helm release name. I think there are a small n of customers who have multiple dagster installs using that, and this could cause two installs to overwrite the secret. Not sure that any of them are celery-k8s users, and I don't think we've ever really promised it.

We could do something like `celeryConfigSecretName: ~`, and default to a prefixed name